### DISCUSSION TITLE
Beginner Requesting Help

### DISCUSSION THREAD

**Discussion by FranticFeline** [eoyilmaz/displaycal-py3#468] [Q&A]
Hello all,

I want to calibrate my laptop's screen to have greater color accuracy when creating digital art, but I have never performed a calibration before, so I know neither how to use the software nor the hardware.

My laptop is a ten-year-old C55-B5295 Toshiba Satellite with, I believe, an LED-backlit LCD screen, and I am aware that its calibration potential may be limited, but it is the only computer that I have at this time.

Anyway, I recently received a Display Plus HL colorimeter from Calibrite, and I need to try it out before the return window closes, in case there are any issues.
 
I can install DisplayCal from the repository of my Linux Mint 22 OS, but after that and plugging in the colorimeter, I am not sure how to proceed. 

Can anyone advise me?

Much thanks.

**Comment by MStraeten** [comment]
https://www.pointsinfocus.com/learning/digital-darkroom/displaycal-and-argyll-cms-quick-start-guide/

**Reply by FranticFeline** [reply]
Hello Martin, 

Thank you for the guide, and apologies for the late reply. 

I originally intended to ask for assistance on the official DisplayCal website, but I was having trouble making an account until today. 

I just submitted a post with questions to that forum, but if no one assists me there, would you be able to help me out instead?

**Comment by vijaysoophie** [comment]
@FranticFeline  Were you able to use Display Plus HL? It is not listed on the list of supported hardware. I am trying to find out which. colorimeter to purchase.

**Reply by FranticFeline** [reply]
Hi, yes, I was able to use it. I don't think that list has been updated.

**Reply by vijaysoophie** [reply]
Thank you. If you could share your experience with the setup that would be helpful.

One more question: Do you think I will be able to directly read the raw output of the sensors using this setup (Display Plus HL + displaycal/ArgyllCMS)? I am doing some color perception research and I want to know the output of the monitor.

**Reply by FranticFeline** [reply]
I am still a beginner, so I'm afraid I'm not very knowledgeable on the matter, but for the average ICC profile, DisplayCal gives sRGB, Adobe RGB, and DCI P3 gamut coverage and volume percentages, ΔE*76 average, maximum, and RMS values, and information like what you see here: https://imgur.com/a/CnliJ4v. Additionally, there is a verification report feature that can provide further details.

You can also read the DisplayCal manual on the website (https://displaycal.net/) and/or create an account to make inquiries on the forums there.

As for my setup, I am assuming you are referring to what I do to prepare my hardware and software before calibration, so I've listed the steps below:

I clean the screen of my device if it's dirty, connect the colorimeter, uncover its lens, place it flush with the middle of the screen, disable any features that alter the native colors of the screen and/or change what it displays (like a screensaver), let the colorimeter sit in place on the screen while the device is on for at least 30 minutes, move everything into a dark room, ensure I have the correct DisplayCal settings selected, and then begin the profiling and calibration.

**Comment by MStraeten** [comment]
You might check https://www.argyllcms.com/doc/ArgyllDoc.html
